src/app.md

```diff
@@ -271,6 +272,54 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-file-skip-dot] #ex-file-skip-dot
+<a name="ex-file-skip-dot"></a>
+##  Traverse directories while skipping dotfiles
+
+Iterate over the entries that are not hidden files or hidden directories. 
+
+[`WalkDir::filter_entry`] descend recursively entries passing the predicate whereas [`std::iter::Iterator::filter`] would by applied to each `DirEntry` even if the parent is a hidden directory. [`WalkDir::depth`] is used to skip the level 0 which is the current directory `"."`.
```

Comment:
how about:
 "descends recursively into entries passing the `is_not_hidded` predicate."

also:
> "would by applied" - > "would be applied"

and in case of:
> [`WalkDir::depth`] is used to skip the level 0 which is the current directory `"."`
i would suggest rephrasing (we are actually avoiding the "skip" here)

Uses [`WalkDir::depth`] to avoid skipping the root directory even if it starts with '.'.